backend/frontend/templates/frontend/progress_details/page.html

```diff
@@ -120,7 +120,7 @@ <h6>Link to your work</h6>
                 </div>
             {% endif %}
 
-            <div id="reviews"></div>
+            <div id="reviews" class="border-2 p-2 rounded-md grid gap-2"></div>
```

Comment:
If this PR is just for the form/modal, then:

- the recently added review doesn't need to show up because it's missing the actual comment
- the review container can be removed, plus it shows up in topics too which is weird